packages-ts/gauntlet-terra-contracts/src/commands/contracts/ocr2/setBilling.ts

```diff
@@ -23,15 +24,16 @@ const makeCommandInput = async (flags: any, args: string[]): Promise<CommandInpu
   return {
     observationPaymentGjuels: billingInfo.observationPaymentGjuels,
     transmissionPaymentGjuels: billingInfo.transmissionPaymentGjuels,
-    recommendedGasPrice: billingInfo.recommendedGasPrice,
+    recommendedGasPriceUluna: billingInfo.recommendedGasPriceUluna,
   }
 }
 
 const makeContractInput = async (input: CommandInput): Promise<ContractInput> => {
   return {
     config: {
-      recommended_gas_price: new BN(input.recommendedGasPrice).toNumber(),
-      observation_payment: new BN(input.observationPaymentGjuels).toNumber(),
+      observation_payment_gjuels: new BN(input.observationPaymentGjuels).toNumber(),
+      transmission_payment_gjuels: new BN(input.transmissionPaymentGjuels).toNumber(),
+      recommended_gas_price_uluna: input.recommendedGasPriceUluna
```

Comment:
This should be of Decimal type (https://github.com/smartcontractkit/chainlink-terra/pull/36/files#diff-7593d1da79e988b58adce267ba96df185c0abebbfc17323f0de5e749e6419134R56)

@archseer Which format should we be applying here?

Reply:
It's a string, you can pass through the value from https://fcd.terra.dev/v1/txs/gas_prices without modifying it

Reply:
This info comes from the RDD, so we can expect it to come in the right format